website/src/pages/subscriptions/create.astro

```diff
@@ -18,150 +18,182 @@ const variantFilter = {
+                            placeholder='Your name for the subscription'
+                        />
+                    </label>
+                </div>
+
+                <div>
+                    <h3 class='mb-2'>Interval</h3>
+                    <p class='mb-2 max-w-xl text-sm text-gray-400'>
+                        How often you want to receive notifications. You can choose between daily, weekly, and monthly.
+                    </p>
+                    <label class='w-full'>
+                        <select class='select select-bordered select-sm w-full max-w-xl'>
+                            <option>daily</option>
+                            <option>weekly</option>
+                            <option>monthly</option>
+                        </select>
+                    </label>
+                </div>
+
+                <div>
+                    <h3 class='mb-2'>Pathogen</h3>
+                    <p class='mb-2 max-w-xl text-sm text-gray-400'>
+                        The pathogen you want to receive notifications for. You can choose between COVID-19 and Flu.
+                    </p>
+                    <label class='w-full'>
+                        <select class='select select-bordered select-sm w-full max-w-xl'>
+                            <option>COVID-19</option>
+                            <option>Flu</option>
+                        </select>
+                    </label>
+                </div>
             </div>
-        </div>
 
-        <div class='divider'></div>
-
-        <div class='mt-2'>Filter</div>
-        <gs-app lapis={getLapisUrl(Organisms.covid)}>
-            <div class='mt-4 grid grid-cols-2 gap-8'>
-                <div class='grid grid-cols-[150px_1fr] gap-4'>
-                    <div class='mt-2 text-right'>Threshold</div>
-                    <div class='flex w-full'>
-                        <label class='ml-2 w-full'>
-                            <select class='select select-bordered w-full'>
-                                <option>Prevalence</option>
-                                <option>Count</option>
-                            </select>
-                        </label>
-                        <label class={`input input-bordered flex items-center gap-2`}>
-                            <input type='number' step={0.1} min={0} max={100} lang='en' class={`grow w-32`} value={0} />
-                            %
-                        </label>
+            <div class='bordered flex flex-col gap-8 rounded-xl border-2 border-gray-200 p-4'>
+                <div class='flex place-content-between items-center'>
+                    <div class='font-bold'>Notification channels</div>
+                    <button class='btn btn-primary btn-sm'>Manage</button>
+                </div>
+
+                <div class='flex flex-col'>
+                    <div class='flex items-center gap-2'>
+                        <input type='checkbox' class='checkbox' />
```

Comment:
My IDE complains that this `input` doesn't have an associated label

Reply:
I left it out, because it messed up the alignment. I now added it, so it covers the whole field. This also nice, so one can click on the text or the symbol.